reference/uitpas.json

```diff
@@ -2863,6 +2866,118 @@
         ],
         "description": "Retrieve information related to a chip number.\n\nIf the response contains `messages`, they MUST be displayed to the end-user.\n\nThe caller of this request must have `CHIP_NUMBERS_READ` permission."
       }
+    },
+    "/card-systems": {
+      "get": {
+        "summary": "Search card systems",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/CardSystemsPaginatedCollection"
+                },
+                "examples": {
+                  "Example": {
+                    "value": {
+                      "totalItems": 0,
+                      "member": [
+                        {
+                          "id": 0,
+                          "name": "string",
```

Comment:
The example is a bit vague with the `"string"` values, can you add some more specific example values? Thanks!

Reply:
Fixed! 👍